src/backend/dx12/src/device.rs

```diff
@@ -1505,8 +1507,8 @@ impl d::Device<B> for Device {
                             return Some(Err(pso::CreationError::Other));
                         }
                     },
-                    InputSlot: attrib.binding as _,
-                    AlignedByteOffset: attrib.element.offset,
+                    InputSlot: i as _,
+                    AlignedByteOffset: 0, // Rebased during the vertex binding remapping
```

Comment:
Wait, so you are force remapping everything to have zero attribute offset? That is quite rough and may cause us to run out of the vertex buffer could.

Reply:
Never happened so far